templates/includes/sidebar.html

```diff
@@ -0,0 +1,4 @@
+<li><a href="#">Home page</a></li>
```

Comment:
The link for 'Home page' is a placeholder (`#`). Consider replacing it with the actual URL path to the home page, such as `{% url 'taxi:index' %}` if using Django's URL template tag.